### ISSUE TITLE
The Walking Dead Season 2 [PCSE00556]

### ISSUE THREAD

**Issue by EZonTheEyes** [Vita3K/compatibility#1101]
# App summary
- App name: The Walking Dead Season 2
- App serial: PCSE00556
- App version: 1.00

# Vita3K summary
- Version: v0.1.6 2957-4343481c
- Build number: v2957-4343481c
- Commit hash: https://github.com/vita3k/vita3k/commit/4343481 
- CPU backend: Dynamic
- GPU backend: OpenGL

                          

# Test environment summary
- Tested by: EZOnTheEyes
- OS: Windows 10
- CPU: Intel i7-6700 @ 3.4GHz
- GPU: Nvidia GeForce RTX 2060
- RAM: 16 GB

# Issues
-Texture Issues (Persistant after several attempts)
-Otherwise seems to be functioning as intended 

# Screenshots
![image](https://user-images.githubusercontent.com/103911396/210034438-11521cbb-0ed6-4508-ae0e-828084b54e7c.png)
![image](https://user-images.githubusercontent.com/103911396/210034474-22fdb691-7c24-460b-8738-54e26fc14d6f.png)
![image](https://user-images.githubusercontent.com/103911396/210034488-37a385f7-afc8-4dff-a7c8-54cfab9423d5.png)

# Log
[TheWalkingDeadS2.zip](https://github.com/Vita3K/compatibility/files/10323108/TheWalkingDeadS2.zip)

# Recommended labels

- Graphics Bugs
- Ingame (-/+?)

**Comment by yaboifungi** [comment]
Yo someone update the compatibility for the walking dead season 2 to playable not ingame+

**Comment by MrSythos** [comment]
> Yo someone update the compatibility for the walking dead season 2 to playable not ingame+

Yeah both season 2 and 1 now seem to be perfectly playable. Just need the label changed.

**Comment by Walidyen** [comment]
Hello, help plz where i can find this version of the wd 2 (pcse00556). It seems like its the only version work on vita3k android version. I search all internet sources found nothing !!

**Comment by github-user1112** [comment]
should this be " The Walking Dead: Season Two  " to match the other one and game header?
